src/Tizen.NUI/src/internal/ItemView.cs

```diff
@@ -64,7 +64,9 @@ public new Tizen.NUI.PropertyArray Layout
         /// </summary>
         [Obsolete("Deprecated in API6; Will be removed in API9.")]
         [EditorBrowsable(EditorBrowsableState.Never)]
+#pragma warning disable CA1716 // Identifiers should not match keywords
```

Comment:
could you add some brief explanation why suppressing warning? (I know currently we are in API9 and this will be removed soon, but some history needs be remained, I think.
and could you block the other warnings also? CA1716, CA1034 and CA1052, these also could be suppressed at the same time. 
thank you.

Reply:
If need to suppress other warning also, we can do as below. 
#pragma warning disable CA1716, CA1034, CA1052 
        public new class Property
#pragma warning restore CA1716,  CA1034, CA1052

Reply:
I thought the above [Obsolete] message can explain why the warning is suppressed.
But do you think some brief explanation is needed more ?

Reply:
Hello, Thank you for your work.
I missed the "Description of Change" where you mentioned about it.
so brief explanation is not needed.
thank you.